pages/base.py

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.page import Page, PageRegion
+
+
+class Base(Page):
+
+    @property
+    def header(self):
+        return Base.Header(self)
+
+    class Header(PageRegion):
+
+        _dropdown_menu_switch_page_locator = (By.CSS_SELECTOR, '.open ul > li a')
```

Comment:
If you add `_root_locator = (By.ID, 'th-global-navbar')` then you'll be able to make sure any elements are beneath this element, rather than searching the entire document.